backend/clients/twilio_client.py

```diff
@@ -169,19 +213,40 @@ def format_to_queue_message(self, values: dict) -> str:
             for i in range(num_media):
                 media_url = values.get(f"MediaUrl{i}", "")
                 media_type = values.get(f"MediaContentType{i}")
-                if media_url:
-                    media.append(media_url)
-                    context.append({"file": media_url, "type": media_type})
+
+                # AUDIO
                 if media_url and media_type == "audio/ogg":
-                    audio_file_path = self.ogg2mp3(
+                    audio_filepath = self.ogg2mp3(
                         audio_url=media_url, message_sid=values["MessageSid"]
                     )
-                    logger.info(f"Audio converted {audio_file_path}")
+                    logger.info(f"Audio converted {audio_filepath}")
                     transcription = self.transcribe_audio(
-                        wav_path=audio_file_path
+                        wav_path=audio_filepath
                     )
                     message_body = transcription if transcription else ""
```

Comment:
Are you sure the audio files are cleaned up properly? If I read this I think that the `.wav` files are left on the file system.

Reply:
Yeah, probably.